src/attr.rs

```diff
@@ -284,6 +297,7 @@ impl Attribute {
 }
 
 #[cfg(feature = "parsing")]
+#[cfg_attr(syn_enable_doc_cfg, doc(cfg(feature = "parsing")))]
```

Comment:
In general, there's not much reason to attach this attribute to non-`pub` items. We probably don't want it here, or on any other non-`pub` items.

Reply:
If `cargo doc --document-private-items` would be used, then this would make a difference. But I don't think, that the additional noise in the code justifies it.